feature_engine/variable_manipulation.py

```diff
@@ -2,33 +2,77 @@
 # License: BSD 3 clause
 # functions shared across transformers
 
+from typing import List, Optional, Union
 
-def _define_variables(variables):
-    # Check that variable names are passed in a list.
-    # Can take None as value
+import pandas as pd
+
+
+def _define_variables(variables: Union[None, List[str]]) -> Union[List[str], None]:
+    """
```

Comment:
_define_variables can take None, a string or a list of strings. Is this what the Union[None, List[str]] is saying?

Reply:
*nit*: `typing.Optional` is more concise than `Union[None, ...]`
https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
You re right @solegalli  and thanks @ChristopherGS for the suggestion.